sim/include/CaloTree.h

```diff
@@ -158,6 +158,7 @@ class CaloTree
   vector<double> m_edepNonIontruth;
   vector<double> m_edepInvtruth;
   vector<double> m_edepbirktruth;
+  std::vector<std::string> m_processtruth;
```

Comment:
change to `vector` instead of `std::vector`? to be consistent with the other ones